test/duct/logger/timbre_test.clj

```diff
@@ -21,13 +22,13 @@
                 ::timbre/println {}}
         logger (::logger/timbre (ig/init config))]
     (is (re-matches
-         #"(?x)\d\d-\d\d-\d\d\ \d\d:\d\d:\d\d\ [^\s]+\ 
```

Comment:
Yes that was an unintentional change caused by my editor.  

I ran the tests at a REPL pushed the PR and then ran `lein test` and noticed the problem.

I think it would be good to make the tests regexes less brittle though, as the addition of a require at the top also changed all the line numbers and broke the regexes.  Perhaps they should check for `/d/d` instead of specific line numbers?